### PR TITLE
docs: update ajax story

### PR DESCRIPTION
**Pull request by Hzunax**
Updated the storybook docs for ajax. Added some more examples.

### CODE REVIEW

File: packages/ajax/stories/index.stories.mdx [hunk 1]
```diff
@@ -48,15 +48,138 @@ ajax.get('./packages/ajax/stories/data.json').then(response => console.log(respo
 
 ### Installation
 
+Using npm:
+
 ```sh
 npm i --save @lion/ajax
 ```
 
+Using yarn:
```
Comment (daKmoR) [inline packages/ajax/stories/index.stories.mdx]: we only do npm... yarn users usually know how to map it 👍

Comment (Hzunax) [reply packages/ajax/stories/index.stories.mdx]: Removed 👍

File: packages/ajax/stories/index.stories.mdx [hunk 2]
```diff
@@ -56,7 +56,120 @@ npm i --save @lion/ajax
...
+  .get('./packages/ajax/stories/data.json')
+  .then(response => {
+    console.log(response.data);
+  })
+  .catch(error => {
+    console.log(error);
+  });
+```
+
+To post data to the server, pass the data as the second attribute in the `POST` request:
```
Comment (jorenbroekema) [inline packages/ajax/stories/index.stories.mdx]: attribute -> argument

**Comment by jorenbroekema** [comment]
Thanks for the contribution! 🤗

**Pull request merged**